api/src/test/java/org/pmiops/workbench/billing/BillingProjectBufferServiceTest.java

```diff
@@ -593,12 +589,14 @@ public void testOutageRecoveryTime() {
       }
     }
 
+    List<ProjectCountByStatusAndTier> projectCounts =
+        billingProjectBufferEntryDao.getBillingBufferGaugeData();
+
     // Sanity check: there shouldn't be any AVAILABLE projects.
-    assertThat(billingProjectBufferEntryDao.getCountByStatusMap().get(BufferEntryStatus.AVAILABLE))
```

Comment:
not really sure why this test is calling the gauge metrics method ... but I updated it to the new version anyway